### PR TITLE
feat(route/gis): Add route

### PR DESCRIPTION
**Pull request by dzx-dzx**
## Involved Issue / 该 PR 相关 Issue

Close #

## Example for the Proposed Route(s) / 路由地址示例

```routes
/gisreportsonline/c/security-challenges/
```

## New RSS Route Checklist / 新 RSS 路由检查表
  
- [ ] New Route / 新的路由
  - [ ] Follows [Script Standard](https://docs.rsshub.app/joinus/advanced/script-standard) / 跟随 [路由规范](https://docs.rsshub.app/zh/joinus/advanced/script-standard)
- [ ] Anti-bot or rate limit / 反爬/频率限制
  - [ ] If yes, do your code reflect this sign? / 如果有, 是否有对应的措施?
- [ ] [Date and time](https://docs.rsshub.app/joinus/advanced/pub-date) / [日期和时间](https://docs.rsshub.app/zh/joinus/advanced/pub-date)
  - [ ] Parsed / 可以解析
  - [ ] Correct time zone / 时区正确
- [ ] New package added / 添加了新的包
- [ ] `Puppeteer`

### CODE REVIEW

**Review by TonyRL — COMMENTED** [review]
Please use namespace `gisreportsonline` instead.

File: lib/routes/gis/index.ts [hunk 1]
```diff
@@ -0,0 +1,60 @@
+import { Route } from '@/types';
+
+import { load } from 'cheerio';
+import { parseDate } from '@/utils/parse-date';
+import cache from '@/utils/cache';
+import ofetch from '@/utils/ofetch';
+
+export const route: Route = {
+    path: '/:path{.*}',
+    categories: ['traditional-media'],
```
Comment (TonyRL) [inline lib/routes/gis/index.ts:10]: This does not belong to `traditional-media`

File: lib/routes/gis/index.ts [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    example: '/gis/c/security-challenges/',
+    parameters: { path: '包含"Reports"页面下的路径' },
+    name: '',
```
Comment (TonyRL) [inline lib/routes/gis/index.ts:13]: Missing route name

File: lib/routes/gis/namespace.ts [hunk 3]
```diff
@@ -0,0 +1,6 @@
+import type { Namespace } from '@/types';
+
+export const namespace: Namespace = {
+    name: 'gisreportsonline',
```
Comment (TonyRL) [inline lib/routes/gis/namespace.ts:4]:
```suggestion
    name: 'GIS Reports',
```

**Review by TonyRL — COMMENTED** [review]
Please use [namespace](https://docs.rsshub.app/joinus/new-rss/start-code#creating-namespace) `gisreportsonline` instead of `gis`

**Comment by dzx-dzx** [comment]
> Please use [namespace](https://docs.rsshub.app/joinus/new-rss/start-code#creating-namespace) `gisreportsonline` instead of `gis`

Wait, are you saying renaming the folder?

**Pull request merged**